### PR TITLE
[Toolchain] remove metaphysics & elan submodules

### PR DESCRIPTION
**Pull request by l2succes**
- Cleans up unused git submodules from reaction (elan)
- Updates `sync-schema` script to use remote metaphysics `schema.graphql` file instead of MP git submodule
- Removes unused `schema.js`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,8 +41,7 @@
     "semantic-release": "semantic-release",
     "start": "yarn storybook",
     "storybook": "node verify-node-version.js && concurrently --raw --kill-others 'yarn relay --watch' 'start-storybook --quiet -s ./public -p 9001'",
-    "sync-colors": "cd externals/elan && git fetch && git checkout origin/master && cp components/lib/variables/colors.json ../../data",
-    "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/master && cp .env.example .env && yarn install && yarn dump-schema -- ../../data",
+    "sync-schema": "curl https://raw.githubusercontent.com/artsy/metaphysics/master/_schema.graphql -o data/schema.graphql; yarn prettier --write --parser graphql data/schema.graphql",
```
Comment (damassi) [inline package.json]: `curl` - nice, keep it simple

**Comment by damassi** [comment]
👍

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 17c3ca6a76a6f409e4fcf990833355e95963362a

https://deploy-preview-1507--artsy-reaction.netlify.com

File: package.json (hunk 1 above)
Comment (orta) [reply package.json]: Yeahhhh, nice one

**Pull request merged**